ios/emerald_mobile_wallet/AppDelegate.m

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```

Comment:
BSD

Reply:
All iOS apps use this... Not sure we can get away with not having it. Generated by the platform but I can rewrite with the same content?